regulatory_compliance/eventgrid.sp

```diff
@@ -137,3 +157,45 @@ query "eventgrid_domain_identity_provider_enabled" {
       azure_subscription sub;
   EOQ
 }
+
+query "eventgrid_topic_local_auth_enabled" {
+  sql = <<-EOQ
+    select
+      a.id as resource,
+      case
+        when disable_local_auth then 'ok'
+        else 'alarm'
+      end as status,
+      case
+        when disable_local_auth then a.name || ' local authentication disabled.'
+        else a.name || ' local authentication enabled.'
```

Comment:
As per the description of the control, I think the condition needs to be reversed, i.e. if local authentication is disabled, then it should be in alarm.